MM/6.4.0/config/media_manager_5/labels/Settings.dcl

```diff
@@ -4351,6 +4351,22 @@ resource configservice_label settings_import_config_editor_itemid {
   ]
 }
 
+resource configservice_label settings_import_config_editor_assetcategory {
+  key = 'SETTINGS_IMPORT_CONFIG_EDITOR_ASSETCATEGORY'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'AssetCateogry'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'AssetKategori'
+      language_id = data.language.danish.id
+    }
```

Comment:
```suggestion
      default_translation = 'Asset category'
      language_id = data.language.english.id
    },
    {
      default_translation = 'Asset kategori'
      language_id = data.language.danish.id
    }
```

Reply:
That does not match the other naming of ItemId in the dropdown.
It is basically an enum value, with a configurable name.
Therefore, I think its better to keep it without spaces.

This is also the name we use in sheets configuration
<img width="875" height="295" alt="image" src="https://github.com/user-attachments/assets/79f3ad42-cf0f-4948-bb34-1565ad642729" />